src/Common/src/TypeSystem/IL/ILImporter.cs

```diff
@@ -116,6 +116,33 @@ private BasicBlock CreateBasicBlock(int offset)
                 basicBlock = new BasicBlock() { StartOffset = offset };
                 _basicBlocks[offset] = basicBlock;
             }
+
```

Comment:
Here we take advantage of "The ordering of the exception clauses in the Exception Handler Table is important.  If handlers are nested, the most deeply nested try blocks shall come before the try blocks that enclose them." ECMA-335